src/uct/ib/base/ib_iface.h

```diff
@@ -254,7 +260,9 @@ typedef ucs_status_t (*uct_ib_iface_set_ep_failed_func_t)(uct_ib_iface_t *iface,
 struct uct_ib_iface_ops {
     uct_iface_internal_ops_t           super;
     uct_ib_iface_create_cq_func_t      create_cq;
+    uct_ib_iface_destroy_cq_func_t     destroy_cq;
     uct_ib_iface_arm_cq_func_t         arm_cq;
+    uct_ib_iface_pre_arm_func_t        pre_arm;
```

Comment:
can we eliminate some of the operations (arm/pre_arm) ? also, there is arm operation on uct_base_iface_t